projects/Numpy2.md

```diff
@@ -0,0 +1,9 @@
+# Numpy
```

Comment:
rename file to `numpy.md`

Reply:
I can't name two projects both as numpy.py, so i am naming it as numpy_main_project